x-pack/plugins/triggers_actions_ui/server/data/routes/indices.ts

```diff
@@ -53,20 +53,26 @@ export function createIndicesRoute(logger: Logger, router: IRouter, baseRoute: s
     }
 
     let aliases: string[] = [];
-    try {
-      aliases = await getAliasesFromPattern(esClient, pattern);
-    } catch (err) {
-      logger.warn(`route ${path} error getting aliases from pattern "${pattern}": ${err.message}`);
-    }
-
     let indices: string[] = [];
+    let dataStreams: string[] = [];
+
     try {
-      indices = await getIndicesFromPattern(esClient, pattern);
+      [aliases, indices, dataStreams] = await Promise.all([
+        getAliasesFromPattern(esClient, pattern),
+        getIndicesFromPattern(esClient, pattern),
+        getDataStreamsFromPattern(esClient, pattern),
+      ]);
     } catch (err) {
       logger.warn(`route ${path} error getting indices from pattern "${pattern}": ${err.message}`);
     }
 
-    const result = { indices: uniqueCombined(aliases, indices, MAX_INDICES) };
+    const result = {
```

Comment:
We might as well change `uniqueCombined()` to take 3 lists now, instead of calling it twice, since it does a sort.

Reply:
Oh my bad, thanks for catching that! Should be fixed in this [commit](https://github.com/elastic/kibana/pull/137584/commits/58cd235a1aa7255b71db81596fbc5193b96fdbeb)